lib/kasket/configuration_mixin.rb

```diff
@@ -91,7 +91,10 @@ def kasket_expires_in(time)
       @kasket_ttl = time
     end
 
-    attr_reader :kasket_ttl
+    def kasket_ttl
+      @kasket_ttl ||= nil
```

Comment:
huh?  just to get it defined for something?  maybe a comment along those lines if so, looks a bit weird as-is.

Reply:
ah, nvm, I get it, for the reference below :man_facepalming: